docs/index.html

```diff
@@ -88,7 +88,7 @@
     <label class="title"><a href="https://github.com/jlongster/prettier">prettier v<span class="version"></span></a></label>
     <label><input type="number" value="80" id="printWidth"></input> printWidth</label>
     <label><input type="number" value="2" id="tabWidth"></input> tabWidth</label>
-    <label><input type="checkbox" id="singleQuote"></input> singleQuote</label>
+    <label>singleQuote <select id="singleQuote"><option value="none">none</option><option value="js">js</option><option value="jsx">jsx</option><option value="all">all</option></select></label>
```

Comment:
I had a look at git history for `docs/prettier.min.js`, and it looks like you've been regenerating it for each release. I'm leaving it as-is in this PR, and if you think it's a good idea, I can add something to the contributing section in the readme about the workflow for the docs.